cmd/psa/verify_as_attester.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package psa
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/apiclient/verification"
+	"github.com/veraison/evcli/common"
+	"github.com/veraison/go-cose"
+	"github.com/veraison/psatoken"
+)
+
+var (
+	attesterClaimsFile *string
+	attesterKeyFile    *string
+	attesterAPIURL     *string
+	attesterNonceSz    *uint
+)
+
+var (
+	attesterVeraisonClient common.IVeraisonClient = &verification.ChallengeResponseConfig{}
+	attesterCmd                                   = NewAttesterCmd(common.Fs, attesterVeraisonClient)
+)
+
+func NewAttesterCmd(fs afero.Fs, attesterVeraisonClient common.IVeraisonClient) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "attester",
+		Short: "Emulate an Attester",
+		Long: `
+	This command implements the "attester mode" of interaction, where the
+	verifier is the protocol challenger.  Here, the nonce is provided by the API
+	server and the PSA attestation token needs to be created on the fly based on
+	the attester's claims and signing IAK.
+	
+		evcli psa verify-as attester \
+		              --api-server=https://veraison.example/challenge-response/v1 \
+		              --claims=claims.json \
+		              --key=es256.jwk \
+		              --nonce-size=32
+	
+	`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := checkNonceSz(attesterNonceSz); err != nil {
+				return err
+			}
+
```

Comment:
Should we not check here for Valid api-server url before passing further down?

Reply:
I think the best place where to check this is when it reaches the apiclient.